Gemfile

```diff
@@ -108,6 +108,7 @@ group :test do
   gem "capybara-screenshot"
   gem "climate_control"
   gem "rails-controller-testing"
+  gem "rspec-retry"
```

Comment:
Just to point out that this will pull the [rspec-retry](https://rubygems.org/gems/rspec-retry) gem from RubyGems.

However, my understanding is that @JR-G's team forked and extended it as [DFE-Digital/rspec-retry](https://github.com/DFE-Digital/rspec-retry).

The other BAT apps are using the fork:
https://github.com/DFE-Digital/apply-for-teacher-training/blob/32a56b8cb18bc052966c862f816781cefdee0df1/Gemfile#L166

I don't have a strong opinion as to whether we should be using the original gem or the DfE fork. The original gem seems to be abandonware, whereas the DfE fork claims to have improved Ruby 3 compatibility. So maybe we should stick with the fork?

Reply:
Ah, thanks for pointing this out. I just did what the docs on the [DFE-Digital/rspec-retry](https://github.com/DFE-Digital/rspec-retry) repo said to do. I'll use the fork instead.